bundles/org.openhab.voice.openaitts/README.md

```diff
@@ -0,0 +1,21 @@
+# OpenAI Text-to-Speech
+
+The OpenAI TTS (Text-to-Speech) addon for openHAB allows you to integrate OpenAI's Text-to-Speech capabilities into your openHAB system. The advantage of this service over others is that one selected voice can speak different languages. This is useful, for example, in conjunction with chatGPT binding, which will help in learning foreign languages.
```

Comment:
It's easier to diff future changes when sentences starts on a new line:

```suggestion
The OpenAI TTS (Text-to-Speech) addon for openHAB allows you to integrate OpenAI's Text-to-Speech capabilities into your openHAB system.
The advantage of this service over others is that one selected voice can speak different languages.
This is useful, for example, in conjunction with chatGPT binding, which will help in learning foreign languages.
```

Reply:
While you're at it:

* addon :arrow_right: add-on
* chatGPT :arrow_right: ChatGTT